src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/PaymentsProcessor.java

```diff
@@ -52,34 +48,30 @@ public void createPayments(Envelope envelope, Long caseId, boolean isExceptionRe
         }
     }
 
-    public void updatePayments(CaseDetails exceptionRecord, String newCaseId) {
-        boolean containsPayments =
-            Objects.equals(
-                exceptionRecord.getData().get(ExceptionRecordFields.CONTAINS_PAYMENTS),
-                YesNoFieldValues.YES
-            );
-
-        if (containsPayments) {
-
-            log.info("Contains Payments, sending payment update message. ER id: {}", exceptionRecord.getId());
+    public void updatePayments(
+        UpdatePaymentsData updatePaymentsData,
+        String exceptionRecordId,
+        String jurisdiction,
+        String newCaseId
+    ) {
+        if (updatePaymentsData.containsPayments) {
```

Comment:
can we plz rename the class`UpdatePaymentsData` to `PaymentsHelper` or something else. Currently it looks payments data contains payments :)

Reply:
Couldn't think of any other name so chose proposed. We can rename laters I guess :)